k8s/solr/README.md

```diff
@@ -0,0 +1,417 @@
+# Overview
+
+Apache Solr is an open-source enterprise-search platform, written in Java, from the Apache Lucene project.
+
+For more information on Apache Solr, see the Apache Solr [official website](https://lucene.apache.org/solr/).
+
+## About Google Click to Deploy
+
+Popular open stacks on Kubernetes, packaged by Google.
+
+## Architecture
+
+![Architecture diagram](resources/apache-solr-k8s-app-architecture.png)
+
+This solution is focused on SolrCloud mode which will use ZooKeeper as external repository. By default 3 replicas of Solr nodes and 3 replicas of ZooKeeper nodes will be deployed. For more information about Solrcloud visit [official documentation](https://lucene.apache.org/solr/guide/8_6/solrcloud.html).
+
+[Basic authentication plugin](https://lucene.apache.org/solr/guide/8_6/basic-authentication-plugin.html) will be enabled for SolrCloud by job resource during post deployment.
```

Comment:
I would put Job as we're talking about a K8s resource.

Reply:
will be enabled -> is enabled